### PR TITLE
Ci/add to workflow & modify dependabot

### PR DESCRIPTION
**Pull request by ErikssonJoakim**
This PR adds the following workflows : 

- `build`
- `publish`
- `release`
- `test`

it also modifies the existing workflows:

- `follow-contributor`: minor action version upgrade
- `lint`: add linter workflow for dockerfile, typescript, json files. There are also some minor changes.

It also adds a `Dockerfile` and adds standalone output.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+    schedule:
+      interval: weekly
+    reviewers:
+      - ccamel
+      - fredericvilcot
+      - ErikssonJoakim
+    assignees:
+      - ccamel
+      - fredericvilcot
+    open-pull-requests-limit: 5
```
Comment (ccamel) [inline .github/dependabot.yml:27]: As you have added a `Dockerfile`, it should also be handled by `Dependabot`. (see: [dependabot#package-ecosystem](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem)).

```suggestion
    open-pull-requests-limit: 5
  - package-ecosystem: docker
    directory: /
    schedule:
      interval: weekly
    open-pull-requests-limit: 2
    reviewers:
      - ccamel
      - fredericvilcot
      - ErikssonJoakim
    assignees:
      - ccamel
      - fredericvilcot
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - build
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.OKP4_TOKEN }}
+
+      - name: Release project
+        uses: cycjimmy/semantic-release-action@v3
```
Comment (ccamel) [inline .github/workflows/release.yml:28]: Will not work until you the `.releaserc.yml` configuration file at the root of the project.

See: 
- https://semantic-release.gitbook.io/semantic-release/usage/configuration
- https://github.com/okp4/ui/blob/main/.releaserc.yml

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      - lint
+      - build
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.OKP4_TOKEN }}
+
+      - name: Release project
```
Comment (ccamel) [inline .github/workflows/release.yml:27]: The GPG keys import is missing for @bot-anik  to sign Git commits.

See: https://github.com/okp4/data-selector/blob/1a1e547855abf1221f0ff82f738e4e3736df7d64/.github/workflows/release.yml#L32

**Review by ccamel — CHANGES_REQUESTED** [review]
Good work. 😌 There are a few remarks about the CI that need to be taken into account for everything to work properly. You can ask @hleonardtek for help.

**Review by hleonardtek — APPROVED** [review]
Looks good ! :+1:

**Comment by ErikssonJoakim** [comment]
@ccamel Thanks Christophe I've added the changes you suggested.

**Review by ccamel — APPROVED** [review]
Great! Thanks :+1:

**Pull request merged**